plugins/outputs/postgresql/README.md

```diff
@@ -0,0 +1,53 @@
+# PostgreSQL Output Plugin
+
+This output plugin writes all metrics to PostgreSQL. 
+The plugin manages the schema automatically updating missing columns, and checking if existing ones are of the proper type. 
+
```

Comment:
Let's document the minimum version of Postgres and TimescaleDB the plugin supports.